### PR TITLE
fix(build): import Flatpickr Locale on demand via regular imports

### PR DESCRIPTION
**Pull request by ghiscoding**
- we shouldn't use require(locale) because they all end up being part of the final bundle, it's better to let the user import whichever Flatpickr Locale he wants to use and that will end up being a much smaller bundle with only the locale we really need

### CODE REVIEW

File: packages/common/src/editors/__tests__/dateEditor.spec.ts [hunk 1]
```diff
@@ -434,10 +434,24 @@ describe('DateEditor', () => {
     });
 
     describe('with different locale', () => {
-      it('should display text in new locale', (done) => {
+      it('should display a console warning when locale it not previously imported', (done) => {
```
Comment (zewa666) [inline packages/common/src/editors/__tests__/dateEditor.spec.ts:437]: Locale it(is)

**Comment by zewa666** [comment]
This is definitely a better approach since you aint really gonna need all locales bundled all the time.

Tbh this actually is a breaking change, because users depending on a different language would now have to touch code as their CI/Tests might break. It feels relatively small but essentially it still is one ;) dunno If you just want to document this which might be enough given the en fallback

**Comment by ghiscoding** [comment]
hmm yeah it might be a small breaking change but I certainly won't do a version 4.x lol
I will document it in Wikis and the Release, to be fair there's a warning message so they will eventually see what to do and there's probably a low number of developers using multiple locales.

**Comment by zewa666** [comment]
Well you asked me and you got an answer ;)
I'm by god no overzealos semver fanatic so I'd tend to go with what you said that the benefit and clear/simple fix should be ok

**Comment by ghiscoding** [comment]
haha I do appreciate your feedback since you're a big user of the lib 😉

**Pull request merged**